content/developer/reference/frontend/javascript_modules.rst

```diff
@@ -51,50 +51,64 @@ execute them precisely in that order).
 Native Javascript Modules
 =========================
 
-Most new Odoo javascript code should use the native javascript module system. This
-is simpler, and brings the benefits of a better developer experience with a better
-integration with the IDE.
-
-There is a very important point to know: Odoo needs to know which files
-should be translated into :ref:`Odoo modules <frontend/modules/odoo_module>` and which
-files should not be translated. This is an opt-in system: Odoo will look at the
-first line of a JS file and check if it contains the string *@odoo-module*. If so, it will
-automatically be converted to an Odoo module.
+Odoo javascript code uses the native javascript module system. This is simpler, and
+brings the benefits of a better developer experience with a better integration with the IDE.
 
-For example, let us consider the following module, located in :file:`web/static/src/file_a.js`:
+Let us consider the following module, located in :file:`web/static/src/file_a.js`:
 
 .. code-block:: javascript
 
-  /** @odoo-module **/
-  import { someFunction } from './file_b';
+  import { someFunction } from "./file_b";
 
   export function otherFunction(val) {
       return someFunction(val + 3);
   }
 
-Note the comment in the first line: it describes that this file should be converted.
-Any file without this comment will be kept as-is (which will most likely be an
-error). This file will then be translated into an Odoo module that look like this:
+There is a very important point to know: by default Odoo transpiles files under
+`/static/src` and `/static/tests` into :ref:`Odoo modules <frontend/modules/odoo_module>`.
+This file will then be transpiled into an Odoo module that look like this:
 
 .. code-block:: javascript
 
-   odoo.define('@web/file_a', function (require) {
-   'use strict';
-   let __exports = {};
+  odoo.define('@web/file_a', ['@web/file_b'], function (require) {
+  'use strict';
+  let __exports = {};
 
-   const { someFunction } = require("@web/file_b");
+  const { someFunction } = require("@web/file_b");
 
-   __exports.otherFunction = function otherFunction(val) {
-       return someFunction(val + 3);
-   };
+  __exports.otherFunction = function otherFunction(val) {
+      return someFunction(val + 3);
+  };
 
-   return __exports;
-   )};
+  return __exports;
+  )};
 
-So, as you can see, the transformation is basically adding `odoo.define` on top,
+So, as you can see, the transformation is basically adding `odoo.define` on top
 and updating the import/export statements.
+This is an opt-out system, it's possible to tell the transpiler to ignore the file.
```

Comment:
Pay attention that RST ignores single line breaks. So if you intend to create a new paragraph, you should add a blank line. If not, it's fine to start the new sentence on the previous line.